plugins/inputs/graylog/README.md

```diff
@@ -0,0 +1,44 @@
+# GrayLog plugin
+
+The Graylog plugin can collect data from remote Graylog service URLs which respond. 
+
+
+Sample configration
+```
+[[inputs.graylog]]
+  ## NOTE This plugin only reads numerical measurements, strings and booleans
+  ## will be ignored.
+
+  ## a name for the service being polled
+  name = "graylog_jvm"
+
+  ## URL of each server in the service's cluster
+  servers = [
+    "http://10.224.162.16:12900/system/metrics/multiple"
+  ]
+
+
+  ## List of tag names to extract from top-level of JSON server response
+  tag_keys = [
+  ]
+
+  metrics = [
+    "jvm.cl.loaded",
+    "jvm.memory.pools.Metaspace.committed"
+  ]
+  ## HTTP Header parameters (all values must be strings)
+  [inputs.graylog.headers]
+    Authorization = "Basic YWRtaW46YWRtaW4"
+    Content-Type = "application/json"
```

Comment:
Why allow users to set these headers? Content-Type and Accept should ALWAYS be application/json. Authorization should be controlled by providing an "auth_key" argument to users, which the plugin would then internally create the Authorization header for the user.